src/node-jspm-jasmine.js

```diff
@@ -88,27 +85,80 @@ export function runTests(opts, errCallback = function() {}) {
 				console.log(`Be sure to provide opts.coverage.files if using the JS API, or --coverage-files <pattern> if using the CLI. Also, be sure to put the pattern in quotes to avoid the glob being expanded before jspmjasmine executes.`);
 			}
 
-			SystemJS.instantiate = (load) => {
-				let normalizedAddress = getOSFilePath(load.address)
-				if (coverageFiles[normalizedAddress]) {
-					load.address = normalizedAddress;
-					coveredModules.push(load.address);
-					if ( load.metadata.sourceMap ) {
-						delete load.metadata.sourceMap.sourcesContent;
-						load.metadata.sourceMap.file = load.address + tranpiledFileSuffix
-						load.metadata.sourceMap.sources = load.metadata.sourceMap.sources.map(
-							filename => getOSFilePath(filename)
-						)
-						fs.writeFileSync(
-							load.address + tranpiledFileSuffix,
-							load.source  + '\n' + inlineSourceMap(load.metadata.sourceMap)
+			// create systemjs hook to allow Istanbul to instrument transpiled sources
+			const instrument = new Instrumenter();
+			const systemInstantiate = SystemJS.instantiate;
+			const tempDirectory = path.join(__dirname, '../no-source-map/');
+			// "instantiate" is the hook that provides the transpiled source
+			SystemJS.instantiate = function(load) {
+				try {
+					// create a unique key to store the sources of modules for the browser
+					const fileKey = getFileKey(getOSFilePath(load.address), process.cwd());
+					// exclude the dependency modules (i.e. libraries) from instrumentation
+					if (coverageFiles[fileKey]) {
+						// put file's transpiled counterpart in temp folder
+						let filename;
+						let sourceMap = '';
+						// arrange sourcemaps
+						if (load.metadata.sourceMap) {
```

Comment:
@joeldenning heh, it's bizarre how Master is coping with it, either errors are being suppressed or parts of your coverage report is missing/inaccurate, or worse, my understanding of the issue is entirely distorted.